### PR TITLE
[BPK-1557]: Amend the docs for the first batch of components

### PR DESCRIPTION
**Pull request by jamesf3rguson**
The following component pages are affected: Accordion, Alignment, Autosuggest, Badge, Banner alert, Bar chart, Blockquote, Breakpoint, Button, Calendar, Card, Chip, Code, Datepicker, and Description List

### CODE REVIEW

File: packages/bpk-docs/src/pages/CardsPage/CardsPage.js [hunk 1]
```diff
@@ -60,14 +60,7 @@ const components = [
...
     title="Cards"
-    blurb={[
-      <Paragraph>
-        Cards are used to group related items within the UI. They allow complex
-        datasets to be broken down into individual, distinct areas for easy
-        consumption. Cards are used as a signifier to give affordance to a
-        clickable set of elements.
-      </Paragraph>,
-    ]}
+    blurb={[]}
```
[the remark this answers is not in the document]
Comment (shaundon) [reply packages/bpk-docs/src/pages/CardsPage/CardsPage.js]: Also as discussed, we should preserve the blurb on the web/native pages for the old docs site (pain I know!)

You can detect if it's the new docs site with `process.env.BPK_NEO`. E.g.:

```
const isNeo = process.env.BPK_NEO;

blurb={
  isNeo ? [] : [<Paragraph>Hi old docs site!</Paragraph>]
}
```

File: packages/bpk-docs/src/pages/NativeCardsPage/NativeCardsPage.js [hunk 2]
```diff
@@ -158,14 +157,7 @@ const components = [
...
     title="Cards"
-    blurb={[
-      <Paragraph>
-        Cards are used to group related items within the UI. They allow complex
-        datasets to be broken down into individual, distinct areas for easy
-        consumption. Cards are used as a signifier to give affordance to a
-        clickable set of elements.
-      </Paragraph>,
-    ]}
+    blurb={[]}
```
Comment (shaundon) [inline packages/bpk-docs/src/pages/NativeCardsPage/NativeCardsPage.js]: Same thing here about selectively removing the blurb.

File: packages/bpk-docs/src/pages/NeoCardPage/CardPage.js [hunk 3]
```diff
@@ -28,7 +28,14 @@ import Native from '../NativeCardsPage';
 const Page = () => (
   <DocsPageWrapper
     title="Neo Card"
-    blurb={[<Paragraph>Blurb for the neo card page.</Paragraph>]}
+    blurb={[
+      <Paragraph>
+        Cards are used to group related items within the UI. They allow complex
```
Comment (shaundon) [inline packages/bpk-docs/src/pages/NeoCardPage/CardPage.js]: The thing we discussed about removing `within the UI` to just make it terser.

File: packages/bpk-docs/src/pages/AutosuggestPage/AutosuggestPage.js [hunk 4]
```diff
@@ -272,14 +271,8 @@ const AutosuggestPage = () => (
...
       <Paragraph>
-        The Backpack autosuggest component is a lightweight wrapper around{' '}
-        <BpkLink href="http://react-autosuggest.js.org/" blank>
-          React Autosuggest
-        </BpkLink>. It can be used to display suggestions for travel
-        destinations, hotels, car hire and more - any data source can be used.
-        Whilst you have full control over suggestion rendering, Backpack
-        provides a built-in suggestion component allowing you to display icons,
-        sub-headings and more.
+        The autosuggest component assist users in query creation and completion
```
[the remark this answers is not in the document]
Comment (jamesf3rguson) [reply packages/bpk-docs/src/pages/AutosuggestPage/AutosuggestPage.js]: ✅

File: packages/bpk-docs/src/pages/BarchartsPage/BarchartsPage.js [hunk 5]
```diff
@@ -139,11 +139,6 @@ const BarchartsPage = () => (
         data. At Skyscanner, bar charts are commonly used for displaying fare
         prices within a given time period e.g. a year, month or week.
       </Paragraph>,
-      <Paragraph>
-        Depending on your need these can be configured to show a variety of
-        details such as gridlines, labels, and axis. They can also be configured
-        to trigger actions on click such as opening a link.
-      </Paragraph>,
```
[the remark this answers is not in the document]
Comment (jamesf3rguson) [reply packages/bpk-docs/src/pages/BarchartsPage/BarchartsPage.js]: ✅

File: packages/bpk-docs/src/pages/BreakpointsPage/BreakpointsPage.js [hunk 6]
```diff
@@ -88,8 +88,10 @@ const BreakpointsPage = () => (
     title="Breakpoints"
     blurb={[
       <Paragraph>
-        To simplify things, Backpack uses only three breakpoints optimised for
-        mobile, tablet and desktop viewports.
+        The Breakpoint component allows information to be displayed in different
```
[the remark this answers is not in the document]
Comment (jamesf3rguson) [reply packages/bpk-docs/src/pages/BreakpointsPage/BreakpointsPage.js]: ✅ lowercase - will add to the decisions file

File: packages/bpk-docs/src/pages/DescriptionListsPage/DescriptionListsPage.js [hunk 7]
```diff
@@ -27,42 +27,48 @@ import descriptionlistsReadme from 'bpk-component-description-list/readme.md';
 
 import DocsPageBuilder from './../../components/DocsPageBuilder';
 import Paragraph from './../../components/Paragraph';
-import PresentationBlock from './../../components/PresentationBlock';
 
 const blurb = [
   <Paragraph>
-    The description list component allows you to create a definition term an
-    relative description.
+    The description list component allows can be used to create a definition
```
[the remark this answers is not in the document]
Comment (jamesf3rguson) [reply packages/bpk-docs/src/pages/DescriptionListsPage/DescriptionListsPage.js]: 🙈 ✅

File: packages/bpk-docs/src/pages/NativeCardsPage/NativeCardsPage.js (hunk 2 above)
Comment (jamesf3rguson) [reply packages/bpk-docs/src/pages/NativeCardsPage/NativeCardsPage.js]: ✅

File: packages/bpk-docs/src/pages/NativeCardsPage/NativeCardsPage.js [hunk 8]
```diff
@@ -155,17 +155,23 @@ const components = [
...
-        consumption. Cards are used as a signifier to give affordance to a
-        clickable set of elements.
-      </Paragraph>,
-    ]}
+    blurb={
+      isNeo
+        ? []
+        : [
+            <Paragraph>
+              - Cards are used to group related items within the UI. They allow
```
Comment (shaundon) [inline packages/bpk-docs/src/pages/NativeCardsPage/NativeCardsPage.js]: Did you copy pasta this from a diff, as there's `-` symbols interspersed in it 😆

File: packages/bpk-docs/src/pages/NeoCardPage/CardPage.js [hunk 9]
```diff
@@ -28,7 +28,14 @@ import Native from '../NativeCardsPage';
 const Page = () => (
   <DocsPageWrapper
     title="Neo Card"
```
Comment (shaundon) [inline packages/bpk-docs/src/pages/NeoCardPage/CardPage.js]: Can you remove `Neo` from the title? (My bad - we discussed it earlier but I forgot to add a comment about it.)

**Comment by jamesf3rguson** [comment]
Thanks @shaundon - I'll do that once the build passes

**Pull request merged**